Slim/Schema/Track.pm

```diff
@@ -29,7 +29,8 @@ our @allColumns = (qw(
 	timestamp added_time updated_time filesize disc remote audio audio_size audio_offset year secs
 	cover cover_cached vbr_scale bitrate samplerate samplesize channels block_alignment endian
 	bpm tagversion drm musicmagic_mixable dlna_profile
-	musicbrainz_id lossless lyrics replay_gain replay_peak extid virtual work subtitle grouping
+	musicbrainz_id lossless lyrics replay_gain replay_peak extid virtual work subtitle grouping performance
```

Comment:
What is `subtitle` vs. `discsubtitle`?

Reply:
subtitle is for the track. discsubtitle is a description for the disc number in multi-disc releases.